persistence/db.go

```diff
@@ -65,31 +68,51 @@ func (pg *PostgresContext) ResetContext() error {
 	return nil
 }
 
-// TECHDEBT: Implement proper connection pooling
-func connectToDatabase(postgresUrl string, schema string) (*pgx.Conn, error) {
+func connectToDatabase(cfg modules.PersistenceConfig) (*pgx.Conn, error) {
 	ctx := context.TODO()
 
-	conn, err := pgx.Connect(context.Background(), postgresUrl)
+	config, err := pgxpool.ParseConfig(cfg.GetPostgresUrl())
+	if err != nil {
+		return nil, fmt.Errorf("unable to create database config: %v", err)
+	}
+	MaxConnLifetime, err := time.ParseDuration(cfg.GetMaxConnLifetime())
```

Comment:
Local var should start with a lowercase letter. Ditto below

Reply:
Yep, shall I raise PR on top of "396-consolidate-config" branch?

Reply:
Yup